src/view/newPoint.js

```diff
@@ -175,27 +176,37 @@ const createNewPointTemplate = (point = {}, isEdit = true) => {
             </li>`;
 };
 
-export default class NewPoint {
-  #element = null;
+export default class NewPoint extends AbstractView {
   #point = null;
 
   constructor(point) {
+    super();
     this.#point = point;
   }
 
-  get element() {
-    if (!this.#element) {
-      this.#element = createElement(this.template);
+  get template() {
+    return createNewPointTemplate(this.#point);
+  }
+
+  setClickDefaultPoint = (callback) => {
+    this._callback.defaultClick = callback;
+    if (this.element.querySelector('.event__rollup-btn')) {
+      this.element.querySelector('.event__rollup-btn').addEventListener('click', this.#defaultPointState);
     }
+    this.element.querySelector('.event__reset-btn').addEventListener('click', this.#defaultPointState);
```

Comment:
```suggestion

    this.element.querySelector('.event__reset-btn').addEventListener('click', this.#defaultPointState);
```

тяжело читать такой код.